data/actions/actions.xml

```diff
@@ -16,7 +16,7 @@
 	<action uniqueid="30015" script="quests/annihilator.lua" />
 
 	<!-- Tools -->
-	<action itemid="2120" script="tools/rope.lua" />
+	<action itemid="2120;7731" script="tools/rope.lua" />
```

Comment:
Let's not modify the default datapack to use this.

Reply:
I've reverted this change.